packages/commerce-sdk-react/src/types.ts

```diff
@@ -44,3 +44,8 @@ export interface Logger {
      */
     error(message: string, args?: Record<string, unknown>): void
 }
+
+export type PasswordlessConfig = {
+    mode?: string
+    callbackURI?: string
+}
```

Comment:
Should this type be more restrictive? Like below (only an exampl)

```
export type PasswordlessConfig = {
    mode?: 'sms' | 'whatever'
    callbackURI?: `/${string}` | `http://${string}` | `https://${string}`
}
```

Ideally these types will be coming from the "isomorphic" lib tho.